gridscale/provider.go

```diff
@@ -41,6 +41,18 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("GRIDSCALE_TF_HEADERS", nil),
 				Description: "Custom HTTP headers",
 			},
+			"request_delay_int": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("GRIDSCALE_TF_REQUEST_DELAY_INT", nil),
+				Description: "Custom request delay interval in Ms. This time interval is used to delay the synchronous request checks, or delay retryable requests.",
```

Comment:
Change `request_delay_int` to `request_delay_interval` and `GRIDSCALE_TF_REQUEST_DELAY_INT` to `GRIDSCALE_TF_REQUEST_DELAY_INTERVAL`, respectively.